src/Common/tests/System/Security/Cryptography/ByteUtils.cs

```diff
@@ -36,11 +37,26 @@ internal static byte[] HexToByteArray(this string hexString)
 
         internal static string ByteArrayToHex(this byte[] bytes)
         {
+            return ByteArrayToHex(bytes.AsReadOnlySpan());
+        }
+
+        internal static string ByteArrayToHex(this Span<byte> bytes)
+        {
+            return ByteArrayToHex((ReadOnlySpan<byte>)bytes);
+        }
+
+        internal static string ByteArrayToHex(this ReadOnlyMemory<byte> bytes)
+        {
+            return ByteArrayToHex(bytes.Span);
+        }
+
+        internal static string ByteArrayToHex(this ReadOnlySpan<byte> bytes)
```

Comment:
We will be adding UTF16 version of https://github.com/dotnet/corefx/blob/master/src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Integer.Unsigned.X.cs#L15 at some point. It would be better to use it, instead of creating all the temporary strings.

Reply:
Yeah, but since this file is only used in tests, it's not a huge deal :)